### ISSUE TITLE
Mammary duct ectasia

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10094]
**Preferred term label:**
Mammary duct ectasia

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0152442

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Mammary duct ectasia

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
added as
HP:0020224
